content/livesync/connector/index.textile

```diff
@@ -0,0 +1,232 @@
+
+When you update the data in your database, you record a change event transactionally in a dedicated "outbox table":https://docs.google.com/document/d/19BJ4oYC3ah4wFH0VS5Qv6qHe3qmRS_PpNLSytTOi9tc/edit#heading=h.r00ywdkts746. The Database Connector detects new changes in the outbox and publishes them as messages on Ably Channels. Using the Models SDK, your client applications subscribe to these change events and update their local state with the new data.
+
+The Database Connector is currently a Docker container image that can run on your own infrastructure. If you want to use an Ably-hosted version, please "reach out":https://docs.google.com/forms/d/e/1FAIpQLSd00n1uxgXWPGvMjKwMVL1UDhFKMeh3bSrP52j9AfXifoU-Pg/viewform.
+
+You can access the Database Connector through a Docker container image, allowing you to run it on your infrastructure in close proximity to your database.
+
+```[sh]
+docker pull ghcr.io/ably-labs/adbc:latest
+```
+
+You can deploy this container image on any cloud infrastructure platform that supports running Docker container images.
+For more information on setting up the required tables in your database, see "Creating the Tables":/livesync/connector/tables.
+
+h2(#local-development). Local Development
+
+In this guide demonstrate the use of "Docker Compose":/https://docs.docker.com/compose/ to spin up a PostgreSQL database and a Database Connector instance on your local machine.
+First, create a docker-compose.yml file with the following contents:
+
+First, create a docker-compose.yml file with the following contents:
+
+```[YAML]
+version: '3'
+
+
+services:
+ adbc:
+   image: ghcr.io/ably-labs/adbc:latest
+   env_file:
+     - adbc.env # load config from env file
+
+
+   ## Uncomment below if you want to load config from your adbc.yaml file,
+   ## which takes precendence over config from the env.
+   # volumes:
+   #   - ./adbc.yaml:/adbc.yaml:ro # mount yaml config file
+
+
+   depends_on:
+     postgres:
+       condition: service_healthy
+   networks:
+     adbc_network:
+      postgres:
+   image: postgres:11-alpine
+   ports:
+     - 5432:5432
+   environment:
+     POSTGRES_USER: postgres
+     POSTGRES_PASSWORD: postgres
+     POSTGRES_DB: postgres
+   healthcheck:
+     test: ["CMD", "pg_isready", "-q", "-d", "postgres", "-U", "postgres"]
+     interval: 2s
+     retries: 30
+   networks:
+     adbc_network:
+   volumes:
+     - adbc_postgres_data:/var/lib/postgresql/data
+
+
+volumes:
+ adbc_postgres_data:
+
+
+networks:
+ adbc_network:
+```
+To configure the Database Connector, you have three options: environment variables, YAML configuration file, or via CLI flags. For more information, see "Specifying Configuration":/livesync/configuration.
```

Comment:
Can we link to the specific heading rather than just the page? In the Spaces docs we can do this with a URI fragment `#` e.g. https://ably.com/docs/spaces/avatar#events